packages/next/server/config-shared.ts

```diff
@@ -184,6 +184,25 @@ export interface ExperimentalConfig {
 
   trace?: {
     serviceName: string
+
+    /**
+     * Patch the Console API to automatically send log lines emitted using it to
+     * the OpenTelemetry endpoint via JSON-HTTP. This isn't required if you're
+     * using a third party logging library.
+     */
+    logging?: {
+      /**
+       * The maximum number of log items to collect before flushing. When logs are
+       * being emitted, if this maximum is reached before the logs are scheduled to
```

Comment:
`before the logs are scheduled to be sent`

What's the schedule? Tick based or timeout based?

Reply:
Great question! Right now the scheduler is tick based if `process.nextTick` is available, or immediate if not. I can clarify in the comment.